syntax = "proto3";

service ImageStreamService {
  rpc StreamImages(stream ImageRequest) returns (stream ImageResponse);
}

message ImageRequest {
  bytes image = 1;
}

message ImageResponse {
  bytes image = 1;
}
